### DISCUSSION TITLE
Question: Ability to embed (in external site) just the links block?

### DISCUSSION THREAD

**Discussion by accessiblepixel** [JulianPrieber/littlelink-custom#121] [Feature requests]
Good day!

Thank you so much for writing LittleLink Custom, it's been working great for me, but I had a question/idea.

I was wondering if it were possible to have (as maybe an extra theme, or url parameter) the ability to just output the block of styled links, rather than the usual default template that users would see?

I've worked quite hard on setting up my links how I'd like them, and I think it'd be great if there were a way to embed them into my website somehow, perhaps with an iframe or some javascript ajax maybe.

If I've put this in the wrong place, please forgive me, I'm not very good at using github!

Kind regards,
Jessica

**Comment by JulianPrieber** [comment]
Hey @accessiblepixel, thank you for the submission!

This is indeed the right place for this.
**This is a dam good idea, I'll add it to the list on: https://github.com/JulianPrieber/littlelink-custom/discussions/49.**

I'm not very good with JS, but an iframe would probably require a change of your vhost config if you're not using Apache or the Docker, so I have to figure something out that works.

Again, thank you for the idea!

All the best,
_Julian_

**Comment by MyBlood** [comment]
@JulianPrieber 

Greetings, dear developer!

I understand correctly that html code embedding is not implemented yet?
